probe/endpoint/connection_tracker.go

```diff
@@ -29,6 +30,9 @@ type connectionTracker struct {
 	flowWalker      flowWalker // Interface
 	ebpfTracker     *EbpfTracker
 	reverseResolver *reverseResolver
+
+	ebpfFailureCount int
```

Comment:
In fact, I think we can remove this altogether and use the zero value of time (https://golang.org/pkg/time/#Time.IsZero ) to reset the failures.